test/ibmq/test_ibmq_job.py

```diff
@@ -308,9 +310,11 @@ def test_retrieve_jobs_start_datetime(self, provider):
                             'greater than or equal to past month: {}'
                             .format(i, job.creation_date(), past_month_str))
 
-    @requires_provider
-    def test_retrieve_jobs_end_datetime(self, provider):
+    @requires_qe_access
+    def test_retrieve_jobs_end_datetime(self, qe_token, qe_url):
```

Comment:
Is there a specific reason to use `requires_qe_access` to activate a provider in the `test_retrieve_jobs_*` test cases, rather than just using `requires_provider`?

Reply:
Yes. Because this sets of tests (`test_retrieve_jobs_end_datetime` etc) require there to be jobs submitted in the previous 2 months. `requires_provider` uses the provider specified via `QE_HGP`, which is new and doesn't have jobs from last 2 months.